core/modules/user/user.install

```diff
@@ -177,6 +177,7 @@ function user_schema_alter(&$schema) {
   // @see https://github.com/backdrop/backdrop-issues/issues/5944#issuecomment-1416900861
   if (backdrop_get_installed_schema_version('user') < 1025) {
     unset($schema['users']['fields']['changed']);
+    watchdog('user', 'Database updates are waiting: a new "changed" column in the users table needs to be created.', WATCHDOG_WARNING);
```

Comment:
Thanks @herbdool. I find the warning a bit confusing, esp. "a new "changed" column in the users table needs to be create."  If I were a site admin looking at this , I would wonder  "do I need to create a new "changed" column??". 

Perhaps just:

`The user module has database schema updates to install. You should run the database update script immediately.`

This mirrors the language in the status page.